components/docker-up/docker-up/main.go

```diff
@@ -122,6 +123,14 @@ func runWithinNetns() (err error) {
 	}
 	args = append(args, userArgs...)
 
+	containerIf := "ceth0"
+	eth0, err := netlink.LinkByName(containerIf)
```

Comment:
I think the name `eth0` get confuse, how about this?
```suggestion
	interface, err := netlink.LinkByName(containerIf)
```

Reply:
`interface` is a reserved word. Changed to `netIface`